scripts/titanic.json

```diff
@@ -0,0 +1,155 @@
+{
+    "archived": "zip",
+    "citation": "https://www.kaggle.com/c/titanic/data [probably have to update this",
```

Comment:
For citation, we can leave it as `""`
The description should be a short sentence.
description: "Titanic passenger data use to predict survival on the Titanic"